src/chrome/chromeUtils.ts

```diff
@@ -312,3 +313,45 @@ export function getUrlRegexForBreakOnLoad(url: string): string {
     const escapedFileName = pathToRegex(fileNameWithoutExtension);
     return '.*[\\\\\\/]' + escapedFileName + '([^A-z^0-9].*)?$';
 }
+
+/**
+ * Checks if a given tcp port is currently in use (more accurately, is there a server socket accepting connections on that port)
+ * @param port The port to check
+ * @param host Optional host, defaults to 127.0.0.1
+ * @param timeout Timeout for the socket connect attempt
+ * @returns True if a server socket is listening on the given port, false otherwise
+ */
+export async function isPortInUse(port: number, host: string = '127.0.0.1', timeout = 400): Promise<boolean> {
+    // Basically just create a socket and try to connect on that port, if we can connect, it's open
+    return new Promise<boolean>((accept, _reject) => {
```

Comment:
I'd rename accept to resolve which is the standard name for that argument

Reply:
Done